openedx/openedx-platform
# fix: error while loading iframe on mfe Unit Outline [FC-0062]

## Description

On the Course unit page after merging the PR with [new iframe for xblocks](https://github.com/openedx/frontend-app-authoring/pull/1375), an issue with infinite loading of the iframe appeared if the unit/xblock has tags. This PR solves the problem with the error that appeared. Addition and processing of the tagging functionality is planned in future PRs.
<img alt="image" width="1324" src="https://private-user-images.githubusercontent.com/93188219/385322911-6985a2b6-e3b3-4707-8af8-2a9bd5de8e03.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE0MzIwOTAsIm5iZiI6MTczMTQzMTc5MCwicGF0aCI6Ii85MzE4ODIxOS8zODUzMjI5MTEtNjk4NWEyYjYtZTNiMy00NzA3LThhZjgtMmE5YmQ1ZGU4ZTAzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTEyVDE3MTYzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBmNTY2NDUzZmZmOTg0ZWQyOGQxOGZmNmVhZDlhOGNkYjU4YTMwY2M3YmE4Njk4ZDRjMmE3ZTEzYWI3OTI3OTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.sJqtHbtAUy8iTcNHuqDhn_kqezxU1eIFAjW_CKIPbTE">
## Testing instructions
1. Switch the edx-platform to the branch of this PR
2. Make sure the `contentstore.new_studio_mfe.use_new_unit_page`
3. Run `tutor dev exec cms openedx-assets build --env=dev` to make sure the updated js is in place
4. While on the master branch of authoring mfe, go to the Course unit page that has the added tags
5. The course xblocks should be rendered inside the iframe
<img alt="image" width="1325" src="https://private-user-images.githubusercontent.com/93188219/385323217-f2957e5a-9975-4548-a5f9-5a957085469b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE0MzIwOTAsIm5iZiI6MTczMTQzMTc5MCwicGF0aCI6Ii85MzE4ODIxOS8zODUzMjMyMTctZjI5NTdlNWEtOTk3NS00NTQ4LWE1ZjktNWE5NTcwODU0NjliLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTEyVDE3MTYzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEzZWM4Y2ZkN2M2ZmRiYTFlYzk2ZWZlMDY4NTFiNmJlYzMxMzAyMjJkZWExMGZkYWJkZDNhNjZhYzVjZDA1ZWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.vI8_pBIjAt9lqWnb7HNO-CPRMQEyRTbGyA29qZR4WL0">
